README.md

```diff
@@ -81,6 +82,100 @@ After setting up your environment, you can verify your setup by sending a test m
   - This will trigger the processing of the specified image file in your application.
   - Verify that the auxiliary files are generated and uploaded to your configured S3 bucket, and ensure that the logs indicate a successful run.
 
+## Submitting a Bulk Ingest Job
+
+This workflow is tailored for efficiently processing large quantities of images stored in an S3 bucket and integrating them into a STAC catalog using AWS services. It is designed to streamline the ingestion process for thousands of images awaiting cataloging.
```

Comment:
nit: technically this is just generating STAC items, not integrating them into a STAC catalog. If we want to align with the scope of DI vs DC, maybe: 
```This workflow is tailored for efficiently processing large quantities of images stored in an S3 bucket and creating STAC items for use in a STAC catalog.```

Reply:
Ignore this - the bulk one does the inserts directly.